Source/WTF/wtf/URL.cpp

```diff
@@ -870,26 +870,23 @@ void URL::dump(PrintStream& out) const
     out.print(m_string);
 }
 
-String URL::strippedForUseAsReferrer() const
+std::tuple<String, bool> URL::strippedForUseAsReferrer() const
```

Comment:
What does the boolean signify? Can we use define a struct with a descriptive name instead?

Reply:
it is indicating whether the strip operation changed the url. I'll change it to struct.